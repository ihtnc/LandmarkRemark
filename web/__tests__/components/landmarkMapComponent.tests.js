import React from 'react';
import { shallow } from 'enzyme';
import LandmarkMap from "@components/landmarkMapComponent";

describe("LandmarkMapComponent", () => {
  test("should match latest snapshot", () => {
    const wrapper = shallow(<LandmarkMap />)
    expect(wrapper).toMatchSnapshot();
  });
});