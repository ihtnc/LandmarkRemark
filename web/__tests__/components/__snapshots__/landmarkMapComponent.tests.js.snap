// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandmarkMapComponent should match latest snapshot 1`] = `<DefaultLoadingContainer />`;
